portfolio/src/main/webapp/comments.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Fetches and posts user comments 
+ */
+ function postComment() {
+	fetch('/data').then(response => response.json()).then((comments) => {
+    comments.forEach((comment) => {
+      createComment(comment); 
+    }); 
+	}); 
+}
+
+function createComment(comment) {
+	const commentsContainer = document.getElementById('comment-display');
+  // create the comment element
+	const commentElement = document.createElement('div');
+  // TODO: CONVERT TIMESTAMP TO READABLE TIME
+	commentElement.innerText = comment.username + " [" + comment.timestamp.toString() + "]\n" + comment.content + "\n"; 
```

Comment:
I'm skeptical of this extra newline.  Are you putting something after this or is it just for extra whitespace?  If it's just extra whitespace you'd want to use CSS for that.